lib/internal/modules/esm/importmap.js

```diff
@@ -0,0 +1,135 @@
+'use strict';
```

Comment:
I know this is pedantic, but could this file be named `import_map.js` to match the style of the others? And in general if we’re using two words let’s always use two words, so `import_map` / `importMap` / `import-map` etc.

Reply:
That’s just the name of the value of the `type` attribute, which has to be all lowercase and without `-` or `_` because there are lots of things in HTML that are case-insensitive and can’t have punctuation; like why it’s `onmousemove` and not `onMouseMove` or `on-mouse-move`. Within the text of the MDN doc it’s always described as “import map”, and the spec defines it as two words: https://html.spec.whatwg.org/multipage/webappapis.html#import-map.

Reply:
I couldn't care less about the file name lol (other than never using uppercase in filenames), changing it now.